staging/src/k8s.io/kube-aggregator/pkg/apiserver/apiserver.go

```diff
@@ -192,8 +196,19 @@ func (c completedConfig) NewWithDelegate(delegationTarget genericapiserver.Deleg
 		egressSelector:           c.GenericConfig.EgressSelector,
 	}
 
-	apiGroupInfo := apiservicerest.NewRESTStorage(c.GenericConfig.MergedResourceConfig, c.GenericConfig.RESTOptionsGetter)
-	if err := s.GenericAPIServer.InstallAPIGroup(&apiGroupInfo); err != nil {
+	rp := apiserverinternalrest.StorageProvider{}
+	apiGroupInfo, err := rp.NewRESTStorage(c.GenericConfig.MergedResourceConfig, c.GenericConfig.RESTOptionsGetter)
```

Comment:
I expected to see this in the kube-apiserver, not in the aggregator.

Reply:
The reason I let the aggregator host it is because the storage version API contains information about all apiservers (kube-apiserver, apiextension apiserver, aggregator apiserver, and custom apiservers). It's analogous to APIService, which is also hosted by the aggrgator. WDYT?

Reply:
> The reason I let the aggregator host it is because the storage version API contains information about all apiservers (kube-apiserver, apiextension apiserver, aggregator apiserver, and custom apiservers). It's analogous to APIService, which is also hosted by the aggrgator. WDYT?

I think this is closer to the idea of webhook admission configuration.  It's provided by the kube-apiserver for consumption and usage by other servers, but aggregation is focused on its single mission